nntrainer/layers/layer_node.cpp

```diff
@@ -284,32 +292,37 @@ nntrainer::Layer *LayerNode::getLayer() {
 void LayerNode::updateInputLayers(const std::string &from,
                                   const std::string &to) {
   auto &input_layers =
-    std::get<std::vector<props::InputLayer>>(*layer_node_props);
+    std::get<std::vector<props::InputConnection>>(*layer_node_props);
   for (auto &input_layer : input_layers) {
-    if (istrequal(input_layer, from)) {
-      input_layer.set(to);
+    if (istrequal(input_layer.get().getName(), from)) {
+      input_layer.set({to, 0});
```

Comment:
How about leave the comment as an TODO to support multi-input tensors for input layers?

Reply:
Right, this can be supported, but nntrainer::InputLayer has a restriction to have only single tensor.
I'll add the comment inside input_layer.h :)

Reply:
Done